lepton.go

```diff
@@ -27,14 +29,16 @@ func init() {
 }
 
 type Lepton struct {
-	SPISpeed    int64  `mapstructure:"spi-speed"`
-	FrameOutput string `mapstructure:"frame-output"`
+	SPISpeed    int64         `mapstructure:"spi-speed"`
+	FrameOutput string        `mapstructure:"frame-output"`
+	FFCPeriod   time.Duration `mapstructure:"ffc-period"`
```

Comment:
We have used seconds in the past but this is okay as long as you have check that the parseing works.